app/agents/voice/automatic/tools/breeze/analytics.py

```
@@ -172,25 +172,25 @@ async def get_breeze_marketing_data(params: FunctionCallParams):
             await params.result_callback(response_json.get("data", {"error": "No data field in response"}))
 
     except httpx.HTTPStatusError as e:
-        logger.error(f"HTTP error calling Breeze Marketing API: {e.response.status_code} - {e.response.text}")
-        await params.result_callback({"error": f"Breeze API error: {e.response.status_code}", "details": e.response.text})
+        logger.error(f"Tool Error: [get_breeze_marketing_data] HTTP error calling Breeze Marketing API: {e.response.status_code} - {e.response.text}")
+        await params.result_callback({"error": f"Tool Error: [get_breeze_marketing_data] Breeze API error: {e.response.status_code}", "details": e.response.text})
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)